### ISSUE TITLE
AirDatePicker for Custom Elements

### ISSUE THREAD

**Issue by Unluckynerd** [t1m0n/air-datepicker#597]
I'm using a custom Element (Web component) for Input. When I try to use AirDatePicker with the Input Web component, it is rendered as Inline Date Picker, even when providing `inline: false` in options.

```
const DatePicker = () => {
  const inputRef = useRef();
  const dp = useRef();

  useEffect(() => {
    dp.current = new AirDatepicker(wrapperRef.current, {
      locale: localeEn,
      inline: true,
      view: 'months',
      minView: 'months',
      dateFormat: 'MM-yyyy',
    });
  }, []);

  return <custom-input ref={inputRef} />;
};
```

Only able to use AirDatePicker only in Native Input element, when I'm using other than input html element it is rendered as inline.

**Comment by Unluckynerd** [comment]
https://github.com/t1m0n/air-datepicker/blob/1a54ee2f5285ff28e30cfa3f3525a7dfc436c0cb/src/datepicker.js#L179

Is there any reason for rendering AirDatePicker as inline for all html elements except input? This behavior becomes particularly problematic when incorporating web components or HTML elements beyond the native input element. It will be really helpful if you can change this behaviour @t1m0n

**Comment by t1m0n** [comment]
It was done for simplicity. When developer uses AirDatepicker on block elements, it will automatically render as inline calendar. Because it makes no sense to display it as a regular calendar with a show/hide functionality, since the user will never be able to open it (as there will be no element to focus on).

If you are using custom element would not it better to initialize AirDatepicker inside this element?

**Comment by Unluckynerd** [comment]
Thanks for the suggestion @t1m0n . I have considered that approach too. Primarily input element is used for datepickers but there could be cases where showing the calendar upon button click. Considering the flexibility for different cases, I have raised this issue.

**Comment by t1m0n** [comment]
Yeah maybe you are right, this logic is unnecessary. I'll think how to improve this. Thanks!
